ml_experiment/metadata/MetadataTable.py

```diff
@@ -70,6 +70,17 @@ def get_configuration_id(self, cur: sqlite3.Cursor, configuration: Dict[str, Val
 
         return res[0]
 
+    def get_configuration(self, cur: sqlite3.Cursor, config_id: int) -> Dict[str, ValueType]:
+        table_name = self.get_table_name()
+        cols = list(self.get_columns(cur))
+        col_str = ', '.join(cols)
+        res = (
+            cur.execute(f"SELECT {col_str} FROM '{table_name}' WHERE id=?", (config_id,))
+            .fetchone()
+        )
+
+        return {cols[i]: res[i] for i in range(len(cols))}
```

Comment:
I like `dict(zip(A,B))`. Should this refactor be part of this pr?

Reply:
Yup, I would make it part of this pr since this is the place it is being added. I typically only punt tech debt clean up when it is in code that is technically outside the pr, or when it adds a significant amount of work to the pr.